### DISCUSSION TITLE
Padlock problem

### DISCUSSION THREAD

**Discussion by vsnreal** [SoCuul/SCInsta#86] [Support] [answered]
I enabled the padlock option, but my iPhone's Face ID is broken. When I try to install SCInsta or BHinsta, it gets stuck on the authentication screen.

**Comment by SoCuul** [comment, marked as answer]
Easiest way would be to delete Instagram and reinstall

**Reply by vsnreal** [reply]
Oh wow, really!? I seriously didn’t think of that, hahaha.

I’ll give this a try now. Thanks a lot for your help!
